tests/integ/test_packaging.py

```diff
@@ -52,7 +52,34 @@ def clean_up(session):
     session.clear_packages()
     session.clear_imports()
     session._runtime_version_from_requirement = None
-    yield
+
+
+@pytest.fixture(autouse=True)
+def get_available_versions_for_packages_patched(session):
+    # Save a reference to the original function
+    original_function = session._get_available_versions_for_packages
+    sentinel_version = "0.0.1"
```

Comment:
The sentinel version will be returned for sktime and scikit_fuzzy, if either of those packages are present.